cirras-underwriting-api/cirras-underwriting-service-api/src/main/java/ca/bc/gov/mal/cirras/underwriting/service/api/v1/impl/CirrasDopYieldServiceImpl.java

```diff
@@ -519,6 +534,9 @@ public DopYieldContract<? extends AnnualField> createDopYieldContract(
 
 				//Calculate and save commodity totals
 				updateDeclaredYieldContractCommodityForage(declaredYieldContractGuid, dopYieldContract, userId);
+				
+				//Calculate and save field rollups
+				updateDeclaredYieldRollupForage(declaredYieldContractGuid, dopYieldContract, userId);
 
```

Comment:
I don't see `updateDeclaredYieldRollupForage` executed in `updateDopYieldContract` method